From 904a1a9c94dc8bbd77347da340a6461d7684b1e9 Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Mon, 3 May 2010 12:56:05 -0300
Subject: [PATCH 11/15] spice-vmc: add nr property

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1272891368-32706-11-git-send-email-alevy@redhat.com>
Patchwork-id: 8970
O-Subject: [RHEL-6 PATCH 10/13] spice-vmc: add nr property
Bugzilla: 576488
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

From: alon.levy <alevy@redhat.com>

BZ: 576488
---
 hw/spice-vmc.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/spice-vmc.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/hw/spice-vmc.c b/hw/spice-vmc.c
index 9b35446..74c9e18 100644
--- a/hw/spice-vmc.c
+++ b/hw/spice-vmc.c
@@ -290,6 +290,8 @@ static VirtIOSerialPortInfo spice_vmc_info = {
     .guest_ready   = spice_vmc_guest_ready,
     .have_data     = spice_vmc_have_data,
     .qdev.props = (Property[]) {
+        DEFINE_PROP_UINT32("nr", SpiceVMChannel, vserport.id,
+                           VIRTIO_CONSOLE_BAD_ID),
         DEFINE_PROP_STRING("name", SpiceVMChannel, vserport.name),
         DEFINE_PROP_END_OF_LIST(),
     },
-- 
1.7.0.3

